### PR TITLE
i want to contribute solution link

### PR DESCRIPTION
**Pull request by huyanh10tin**
```
This is a PR template, If you are adding a solution link to the curriculum leave this as is. If you are not, then delete it and write the message you wish.
Thank you,
The Odin Project team.

```

### CODE REVIEW

File: ruby/project_oop.md [hunk 1]
```diff
@@ -354,6 +354,9 @@ Build a Mastermind game from the command line where you have 12 turns to guess t
 * [Neil Cudden's solution](https://github.com/ncud4bloc/Mastermind/)
 * [Roland Studer's solution (with AI)](https://github.com/RolandStuder/odin_project_solutions/tree/master/oop/mastermind)
 * [Iceron's solution (with AI)](https://github.com/iceron/TOP/blob/master/Ruby/oop/mastermind/mastermind.rb)
+* [HuyAnh's solution (no AI)](https://github.com/huyanh10tin/mastermind)
+
+
```
Comment (KevinMulhern) [inline ruby/project_oop.md]: Can you remove these two newlines please?

File: ruby/project_oop.md [hunk 2]
```diff
@@ -190,7 +190,7 @@ Build a tic-tac-toe game on the command line where two human players can play ag
 * [Roland Studer's solution (with slow learning AI)](https://github.com/RolandStuder/odin_project_solutions/tree/master/oop/tic-tac-toe)
 * [Iceron's solution](https://github.com/iceron/TOP/blob/master/Ruby/oop/tic_tac_toe/tic_tac_toe.rb)
 * [Anistor86's solution](https://github.com/anistor86/tic_tac_toe)
-
+* [HuyAnh's solution](https://github.com/huyanh10tin/tictactoe)
```
Comment (KevinMulhern) [inline ruby/project_oop.md]: Can you add a new line below this please?

File: web_development_101/project_pairing.md [hunk 3]
```diff
@@ -46,7 +46,7 @@ Add your solution below this line!
 * [Anistor86 & Isashimitsui's Solution](https://github.com/anistor86/pomodoroclock) | [View in Browser](https://anistor86.github.io/pomodoroclock/)
 * [coryparham24's Solution](https://github.com/coryparham24/pomodoro-clock) | [View in Browser](https://cdn.rawgit.com/coryparham24/pomodoro-clock/7b1e6a88/index.html)
 * [Oliver Curting's Solution](https://github.com/Curting/pomodoro) | [View in Browser](https://curting.github.io/pomodoro/)
-
+* [HuyAnh10tin's Solution](https://github.com/huyanh10tin/ThePomodoro) | [View in Browser](https://huyanh10tin.github.io/ThePomodoro/index.html)
```
Comment (KevinMulhern) [inline web_development_101/project_pairing.md]: Can you add a new line below this too please?

File: ruby/project_oop.md (hunk 2 above)
Comment (huyanh10tin) [reply ruby/project_oop.md]: ok

**Comment by KevinMulhern** [comment]
Hey @huyanh10tin, you only fixed one of the problems?

**Pull request closed without merging**